db/python/tables/analysis.py

```diff
@@ -413,12 +414,13 @@ async def get_sample_cram_path_map_for_seqr(
     ) -> List[dict[str, str]]:
         """Get (ext_sample_id, cram_path, internal_id) map"""
 
-        values: dict[str, Any] = {'project': project}
+        values: dict[str, Any] = {'project': project, 'PRIMARY_EXTERNAL_ORG': PRIMARY_EXTERNAL_ORG}
         filters = [
             'a.active',
             'a.type = "cram"',
             'a.status = "completed"',
-            'p.project = :project',
+            'peid.project = :project',
+            'peid.name = :PRIMARY_EXTERNAL_ORG',
```

Comment:
At least for now, maintained the status quo by only returning a row for the primary external id rather than several, one for each external id. This is a technique I used in several of the “higher level” queries, and will want to be revisited eventually.